modules/core/src/lib/fetch/fetch-file.js

```diff
@@ -16,7 +16,7 @@ export async function fetchFile(url, options) {
   url = resolvePath(url);
   // TODO - SUPPORT reading from `File` objects
   const response = await fetch(url, options);
-  if (!response.ok && options.throws) {
+  if (!response.ok && options && options.throws) {
```

Comment:
I guess we normally default `options = {}`. This is fine too, though since we do pass this parameter to `fetch` (which probably doesn't explicitly say that options can be `undefined`), it may be better to use the default param version?